definitions/mocha_v2.4.x/flow_>=v0.22.x/mocha_v2.4.x.js

```diff
@@ -0,0 +1,19 @@
+type TestFunction = (() => void | Promise) | ((done : Function) => void);
```

Comment:
Also, can we type the `done` callback as `() => void` instead of `Function`? `Function` is an `any` type that is best to avoid when possible